tests/cases/fourslash/organizeImportsGroup_MultiNewlines.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path="fourslash.ts" />
+
+////import c from "C";
+////
+////

```

Comment:
Thanks for the test case. Do we need to be ensuring that we don't end up with too many empty lines, or is that out of scope for the organizer? Probably never had to deal with this before because it merged groups into one.

Reply:
agree, I think it is better if we merge multi newlines into one :D

Reply:
hi, after checking the current organizer, I think we haven't supported collapsing consecutive new lines.
For example, if you organize in vscode, it doesn't remove empty newlines
```js
import {
    a, b,


    c
} from "something";

console.log(a, b, c);
// nothing happen when sorting
```

```js
const something;


import * as a from "a";

import * as b from "b";


import * as c from "c";


import * as d from "d";

console.log(a, b, c, d)
// ----->
const something;


import * as a from "a";
import * as b from "b";
import * as c from "c";
import * as d from "d";






console.log(a, b, c, d)
```

Reply:
hi @jakebailey, do you think if we should merge this pr and create another task for removing consecutive newlines?

Reply:
@RyanCavanaugh @DanielRosenwasser What's your take? I feel like it's awkward to not remove the extra empty lines on organize (lest we leave a lot behind), but maybe that's not required.

Reply:
I'm OK punting it; this seems like an edge case and the user workaround is trivial